app/assets/javascripts/host_edit.js

```diff
@@ -159,6 +159,7 @@ function update_capabilities(capabilities) {
 var stop_pooling;
 
 function submit_with_all_params() {
+  var fqdn = $('.fqdn')[0].textContent;
```

Comment:
This is wrong, I've been reading it incorrectly and it is quite nasty one: https://projects.theforeman.org/issues/34300.